tox.ini

```
@@ -2,9 +2,9 @@
 min_version =  4.11
 
 env_list =
-    python{3.8,3.9,3.10}-dj{32}-wagtail{41,52}
-    python{3.11}-dj{41,42}-wagtail{52,60}
-    python{3.12}-dj{42,50}-wagtail{52,60}
+    python{3.8,3.9,3.10,3.11,3.12}-django3.2-wagtail5.2
```

considering Django 3.2 is soon EOL, I don't think we should test with the full suite of python versions

```suggestion
    python{3.8,3.9,3.10}-django3.2-wagtail5.2
```